src/components/Target.vue

```diff
@@ -20,7 +20,7 @@ export default {
     getCurrentTime: (x) => {x.now = Math.floor(new Date().getTime() / 1000)}
   },
 
-  props: ["val", "buslist"],
+  props: ["buslist"],
```

Comment:
To remove 
![image](https://user-images.githubusercontent.com/27392567/69538264-3fb4aa00-0fc5-11ea-8e8a-b1bcf854b473.png)
this into 
![image](https://user-images.githubusercontent.com/27392567/69538143-fb290e80-0fc4-11ea-87dc-1f40bd7f0e04.png) 
this, you should not just delete the properties. You also need to delete the lines that the prop is used.